cl_sii/libs/rows_processing.py

```diff
@@ -122,36 +122,22 @@ def rows_mm_deserialization_iterator(
             row_data.pop(_field_name, None)
 
         try:
-            mm_result: marshmallow.UnmarshalResult = row_schema.load(row_data)
-            deserialized_row_data: dict = mm_result.data
+            deserialized_row_data: dict = row_schema.load(row_data)
             raised_validation_errors: dict = {}
-            returned_validation_errors: dict = mm_result.errors
         except marshmallow.ValidationError as exc:
             deserialized_row_data = {}
             raised_validation_errors = dict(exc.normalized_messages())
-            returned_validation_errors = {}
 
         validation_errors = raised_validation_errors
-        if returned_validation_errors:
-            if row_schema.strict:
-                # 'marshmallow.schema.BaseSchema':
-                # > :param bool strict: If `True`, raise errors if invalid data are passed in
-                # > instead of failing silently and storing the errors.
-                logger.error(
```

Comment:
That log was removed because the `load` function is not returning the errors anymore, that is why the var `returned_validation_errors` was also removed.

If an error is triggered during the `load` it will raise a `marshmallow.ValidationError` and those error messages are stored in `raised_validation_errors` then logged with `logger.fatal` (as was before) bellow